pkg/apis/terraform/v1alpha1/constraint_types.go

```diff
@@ -45,6 +45,10 @@ type ModuleConstraint struct {
 	// be allowed to run.
 	// +kubebuilder:validation:Optional
 	Allowed []string `json:"allowed,omitempty"`
+	// Selector is the selector on the namespace or labels on the configuration. By leaving this
+	// fields empty you can implicitedly selecting all configurations.
```

Comment:
`By leaving this fields empty you can implicitedly selecting all configurations.` to `By leaving this field empty you are implicitly selecting all configurations.`